doc/loanrules.md

```diff
@@ -1,19 +1,22 @@
 # Loan Rules
 
-The loan rules engine calculates the loan policy (that specifies the loan period)
-based on the patron's patron group and the item's material type, loan type, and location.
+The loan rules engine calculates the loan policies based on the patron's
+patron group and the item's material type, loan type, and location.
 
 Example loan rules file:
 
-    fallback-policy: no-circulation
-    m book : regular-loan
-    m newspaper: reading-room
-    m streaming-subscription: policy-s
-        g visitor undergrad: in-house
+    fallback-policy: l no-circulation
+    fallback-policy: r no-requests
+    fallback-policy: n no-notices
```

Comment:
Sorry, the multi-line fallback syntax is wrong. We don't know how this came into https://issues.folio.org/browse/UICIRC-165
I've added a comment to that issue about the correct single line fallback section.